webrender/res/cs_clip_box_shadow.glsl

```diff
@@ -60,11 +77,22 @@ void main(void) {
 #ifdef WR_FRAGMENT_SHADER
 void main(void) {
     vec2 local_pos = vPos.xy / vPos.z;
+    vec2 uv;
+
+    switch (vStretchMode) {
+        case MODE_STRETCH: {
+            uv = clamp(vUv.xy, vec2(0.0), vEdge.xy);
+            uv += max(vec2(0.0), vUv.xy - vEdge.zw);
+            uv = mix(vUvBounds_NoClamp.xy, vUvBounds_NoClamp.zw, uv);
```

Comment:
nit: could go straight to `vUvBounds` and move the follow up clamping into the `MODE_SIMPLE` case

Reply:
It's not quite clear to me what you mean here, since that statement is a ```mix```. Could you expand on the above?

Reply:
on a second thought, ignore that :)